### PR TITLE
update limit dc num

### PR DESCRIPTION
**Pull request by zhouying7780**

### CODE REVIEW

File: deployment/tc532411/history-limit-dc.yaml [hunk 1]
```diff
@@ -30,4 +30,4 @@ spec:
         - "100"
   triggers:
   - type: ConfigChange
-  revisionHistoryLimit: 3
+  revisionHistoryLimit: 2
```
Comment (wsun1) [inline deployment/tc532411/history-limit-dc.yaml]: Why update to 2?

Comment (zhouying7780) [inline deployment/tc532411/history-limit-dc.yaml]: To reduce the deploy times .

Comment (wsun1) [inline deployment/tc532411/history-limit-dc.yaml]: Why reduce the deploy times?

Comment (zhouying7780) [inline deployment/tc532411/history-limit-dc.yaml]: Use this file to test the function about auto clean the old RC, if revisionHistoryLimit is 3, we need to deploy 5 times to clean the first RC. if revisionHistoryLimit is 2, we only deploy 4 times.

**Pull request merged**